pkgs/development/compilers/avian/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, zlib, openjdk }:
+
+stdenv.mkDerivation {
+  name = "avian-1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "readytalk";
+    repo = "avian";
+    rev = "72c0547f4bdedc1ba8da2882317e3f758deab23f";
+    sha256 = "1j2y45cpqk3x6a743mgpg7z3ivwm7qc9jy6xirvay7ah1qyxmm48";
+  };
+
+  buildInputs = [
+    zlib
+    openjdk
```

Comment:
avian builds fine with oraclejdk as well, so it's better to use `jdk` instead of `openjdk` to allow users override the it (it defaults to `openjdk`)